content/en/docs/contribute/style/diagram-guide.md

```diff
@@ -0,0 +1,447 @@
+    class A,B,C,D box
+
+{{< /mermaid >}}
+
+<figure style="padding-left: 310px; font-weight: bold" alt="diagram guide mermaid+svg method">
+<figcaption><a href="https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb2RlIjoiZ3JhcGggTFJcbiAgICBBWzEuIFVzZSBsaXZlIGVkaXRvcjxicj4gdG8gY3JlYXRlL2VkaXQ8YnI-ZGlhZ3JhbV0gLS0-XG4gICAgQlsyLiBTdG9yZSBkaWFncmFtPGJyPlVSTCBzb21ld2hlcmVdIC0tPlxuICAgIENbMy4gR2VuZXJhdGUgLnN2ZyBmaWxlPGJyPmFuZCBkb3dubG9hZCB0bzxicj5pbWFnZXMvIGZvbGRlcl0gLS0-XG4gICAgRFs0LiBSZWZlcmVuY2UgLnN2ZyBmaWxlPGJyPnVzaW5nIG1hcmtkb3duIGltYWdlPGJyPnN0YXRlbWVudCBpbiBwYWdlPGJyPm1hcmtkb3duIGZpbGUgXVxuXG5cbiAgICBjbGFzc0RlZiBib3ggZmlsbDojZmZmLHN0cm9rZTojMDAwLHN0cm9rZS13aWR0aDoxcHgsY29sb3I6IzAwMDtcbiAgICBjbGFzcyBBLEIsQyxELEUgYm94XG4gICAgIiwibWVybWFpZCI6IntcbiAgXCJ0aGVtZVwiOiBcImRlZmF1bHRcIlxufSIsInVwZGF0ZUVkaXRvciI6ZmFsc2UsImF1dG9TeW5jIjp0cnVlLCJ1cGRhdGVEaWFncmFtIjpmYWxzZX0">Figure x. Mermaid+SVG method steps</a></figcaption><br>
+</figure>
+
+For example, use the live editor to create a diagram called `boxnet`. Store the diagram URL somewhere for later reference. Generate and download a `boxnet.svg` file to the appropriate `../images/` folder. Then reference the `.svg` file in the PR `.md` file using the standard markdown image syntax.
+
+```json
+![my boxnet diagram](static/images/boxnet.svg)
+```
+
+You can also use the [Hugo `{{</* figure */>}}` shortcode](https://gohugo.io/content-management/shortcodes/#figure) to reference a diagram. It allows you to include a title, caption, various link URLs and styling associated with your diagram. 
+
+```
+{{</* figure src="/static/images/boxnet.svg" title="Boxnet figure" caption="Picture of Boxnet" */>}}
+```
+
+
+
+You should add the live editor URL as a comment block in the `.svg` file using a text editor. For example, you would include the following at the beginning of the `.svg` file:
+
+```
+<!-- To view or edit the mermaid code, use the following URL: -->
+<!-- https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb ... -->
+```
+
+The advantages of the Mermaid+SVG method consist of the following:
+
+* Live editor tool.
+* Live editor tool supports the most current Mermaid feature set.  
+* Employ existing K8s/website methods for handling `.svg` file.
+* Environment doesn't require Mermaid support.
+
+Be sure to check the diagram renders properly using the local and Netlify previews.
+
+
+**3. External Tool** - Use your external tool to create a diagram. Save the diagram coordinates for contributor access. For example, your tool may offer a link to the diagram image, or you could place the source code file, such as an `.xml` file, in a public repository for contributor access.   
+
+Generate an `.svg` or `.png` file, download this file to the appropriate `../images/` folder, and then reference the `.svg` or `.png` file using a markdown image statement.
+
+Here is an example markdown image statement for `your-image.svg`. 
+
+```json
+![your-image.svg](static/images/boxnet.svg)
+```
```

Comment:
If it's a diagram, I'd prefer that people use a [`figure`](https://gohugo.io/content-management/shortcodes/#figure) shortcode and that they additionally set a size class.
For example:
```markdown
{{< figure src="/images/docs/pod.svg" alt="Pod creation diagram" class="diagram-medium" >}}
```

Allowed sizes are `small`, `medium` or `large`.